kibbeh/src/ui/BaseSettingsItem.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactNode } from "react";
+
+export interface BaseSettingsItemProps {
+  children: ReactNode;
+}
+
+export const BaseSettingsItem: React.FC<BaseSettingsItemProps> = ({
+  children,
+  ...props
+}) => {
+  return (
+    <div className="bg-primary-900 rounded-8" {...props}>
```

Comment:
Can you add `className` as a prop?
```suggestion
    <div className={`bg-primary-900 rounded-8 ${className}`} {...props}>
```